ui/tests/acceptance/secure-variables-test.js

```diff
@@ -119,6 +119,25 @@ module('Acceptance | secure variables', function (hooks) {
     assert.notOk(fooLink, 'foo0 file is no longer present');
   });
 
+  test('it does not allow you to save if you lack Items', async function (assert) {
+    defaultScenario(server);
```

Comment:
`question`: Should this move into a `beforeEach` hook?

Reply:
Hm not yet; I imagine there will be scenarios for this form where I don't want defaultScenario(), but set up some something custom / edge-casey.